packages/core/utils/lib/errors.js

```diff
@@ -27,6 +27,7 @@ class YupValidationError extends ValidationError {
     super();
     const { errors, message: yupMessage } = formatYupErrors(yupError);
     this.message = message || yupMessage;
+    if(errors.length > 0) this.message += ` (${errors.map(({message}) => `'${message}'`).join(', ')})`
```

Comment:
This error is used to return validation errors in the API. We put all the details in the details key for this reason and need to keep the message basic. 

We might want to use a different error type for your usecase. wdyt?